_config.yml

```diff
@@ -11,7 +11,7 @@ footer_text: >
   Photos from <a href="https://unsplash.com" target="_blank">Unsplash</a>.
 
 url:  # the base hostname & protocol for your site
-baseurl: /al-folio # the subpath of your site, e.g. /blog/
+baseurl: # the subpath of your site, e.g. /blog/
```

Comment:
Please keep it `/al-folio`, so that I can use https://alshedivat.github.io/al-folio/ for the demo webpage.